integration-tests/tasks/konflux-e2e-tests-task.yaml

```diff
@@ -58,6 +58,9 @@ spec:
     - name: e2e-timeout
       description: "Timeout value for running E2E script (cluster bootstrap + E2E tests)"
       default: "90m"
+    - name: enable-sl-plugin
+      description: "Enable Sealights ginkgo plugin to scan the e2e tests that will report to Sealights."
+      default: "false"
```

Comment:
is this actually used somewhere in the code?

Reply:
https://github.com/konflux-ci/e2e-tests/pull/1547/files#diff-c3b08c8e916c3934cb5fb60ee271d2cedd68abd18d6da852f19415ec5e1bdf59R130

Here, Im trying to diferentiate between uploading via apis and now with the plugin